### DISCUSSION TITLE
something strange I do not understand..

### DISCUSSION THREAD

**Discussion by Rinus-Verdonschot-UB** [asreview/asreview#1838] [Q&A] [answered]
I have a fully annotated dataset for which I am checking whether ASReview could have substantially lowered screening time.
According to most of my simulations (random N=5 for incl/excl; random N=10 for incl/excl; 1% SAFE simulation etc) all articles would have been found by 60% of the screening.

![afbeelding](https://github.com/user-attachments/assets/bfe2a5b9-3d26-4b90-b489-f31fdd9cfe36)

I then took the articles which eventually went to full text screening (and were included in the paper) to see roughly where they were included (+ranking) in the screening process.

The way I did that is to run multiple simulations (same seed and priors) and used different stopping points corresponding to % of the total N of the dataset using the --stop_if command. So, for example, stop_if 388 would have been 10% of the dataset, --stop_if 776 would have been 20% of the dataset etc.. and for each simulation i check whether the articles which eventually went into the review were in it.

HOWEVER, when I do this.. i see that the last two articles were included somewhere between 60% and 70% of the simulation..  (I ran this simulation a couple of times with different seed/priors) but results are the same.

So, there is a discrepancy with the full simulation showing all included articles found around 60% (judging from the figure) but the last two FT articles between 60 and 70%

![afbeelding](https://github.com/user-attachments/assets/a4965e01-cfe8-4808-8405-c419e62f471e)

I would appreciate any insights you could give me.

**Edited**: Just for clarification.. the FT articles (N=20) which are the articles which eventually went into the actual review are of course a subset of all "included" articles which passed initial screening (in other words the total number of included in initial screening of dataset of 3878 abstracts was 110 and from these 110 eventually these 20 went into the review)

**Comment by Rensvandeschoot** [comment, marked as answer]
Thank you for your detailed report! 
You might be interested in the Time to Discovery, available in [ASReview-Insights](https://github.com/asreview/asreview-insights?tab=readme-ov-file#time-to-discovery). It will provide you with a list of all the relevant papers when these are found in the labeling process of the simulation study. If you use [ASReview-Makita](https://github.com/asreview/asreview-makita/) you will get a table with all [record ID's](https://asreview.readthedocs.io/en/latest/example_api_asreview_file.html) and you can identify which ones are the full-text relevant ones, see for an example the screenshot. 

![image](https://github.com/user-attachments/assets/0c24e885-2b02-4ed3-8f83-6dfdc930611b)

**Comment by Rensvandeschoot** [comment]
And you might want to read this paper where we used the Time to Discovery to identify how difficult to find papers behave when using a different model: https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-024-02587-0 and mostly the plot with rank orders: https://systematicreviewsjournal.biomedcentral.com/articles/10.1186/s13643-024-02587-0/figures/1
